README.md

```diff
@@ -175,18 +175,77 @@ Venafi secrets engine:
        url="https://tpp.venafi.example" \
        access_token="tn1PwE1QTZorXmvnTowSyA==" \
        refresh_token="MGxV7DzNnclQi9CkJMCXCg==" \
+       refresh_token_2="p0WTt3sDPbzm2BDIkoJROQ==" \
+       zone="DevOps\\HashiCorp Vault" \
+       trust_bundle_file="/opt/venafi/bundle.pem"
+   Success! Data written to: venafi-pki/venafi/tpp
+   ```
+   
+   In order to set the refresh tokens you will need to get two token key pair. For example,
+   you can execute `vcert getcred` command twice as follows:
+   
+   ```
+   $ vcert getcred ...
+   OUTPUT:
+   vCert: 2023/01/21 12:05:30 Getting credentials...
+   access_token:  access_token_1
+   access_token_expires:  ...
+   refresh_token:  refresh_token_1
+   refresh_until:  ...
+   ```
+   
+   Then one more time:
+
+   ```
+   $ vcert getcred ...
+   OUTPUT:
+   vCert: 2023/01/21 12:05:32 Getting credentials...
+   access_token:  access_token_2
+   access_token_expires:  ...
+   refresh_token:  refresh_token_2
+   refresh_until:  ...
+   ```
+   
+   Now you can set the Venafi secret as follows (`access_token_1` and `access_token_2` are interchangeable):
```

Comment:
I dont think this statement is true ?  Can we just be specific here.
Set the 1st pair of accesstoken and refresh token as vault  access token and refresh token.
and set refresh token from 2nd pair to refresh token 2....

Reply:
Got it